enhancements/split-sdk-into-multiple-projects.md

```diff
@@ -0,0 +1,190 @@
+---
+title: split-sdk-into-multiple-projects
+authors:
+  - "@jesusr"
+reviewers:
+  - "@estroz"
+  - "@hasbro17"
+  - "@joelanford"
+  - "@fabianvf"
+  - "@shawn-hurley"
+approvers:
+  - "@estroz"
+  - "@hasbro17"
+  - "@joelanford"
+  - "@fabianvf"
+  - "@shawn-hurley"
+creation-date: 2020-05-18
+last-updated: 2020-05-27
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Split SDK into multiple projects
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions [optional]
+
+1. How will separate repos affect the SDK website and docs in general?
+   * What if each individual repo maintains their own docs? Then have a
+     pipeline sort of thing that would regenerate the entire doc if there are
+     any changes
```

Comment:
We could build a mechanism to add docs from Ansible/Helm repos into the main sdk site at build time. This would keep the top level site as it is now. A drawback would be that the documentation versioning would be for the entire SDK, but it seems like the Ansible operator and its docs should be versioned separately. @ecordell posed a similar plan for pulling in all operator-framework docs into a single repo at build time, which could be used to version the framework as a single entity.
We have a few options, but they each have their drawbacks.
1. We could have separate sites for the ansible/helm sdks, ie ansible.operatorframework.io. This would allow the adapter operators to version their own docs independently, and would mean that docs and code changes could happen in the same PR.